packages/meditrak-server/src/routes/mapOverlayGroupRelations/GETMapOverlayGroupRelations.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { GETHandler } from '../GETHandler';
+import { assertAnyPermissions, assertBESAdminAccess } from '../../permissions';
+import {
+  assertMapOverlayGroupRelationsGetPermissions,
+  createMapOverlayGroupRelationDBFilter,
+  createRelationsViaParentOverlayGroupDBFilter,
+} from './assertMapOverlayGroupRelationsPermissions';
+import { assertMapOverlayGroupsGetPermissions } from '../mapOverlayGroups';
+
+/**
+ * Handles endpoints:
+ * - /mapOverlayGroupRelations
+ * - /mapOverlayGroupRelations/:mapOverlayGroupRelationId
```

Comment:
Should we also add `/mapOverlayGroups/:parentRecordId/mapOverlayGroupRelations`?